src/ppx/application.cpp

```diff
@@ -937,14 +937,29 @@ int Application::Run(int argc, char** argv)
     }
     mStandardOptions = mCommandLineParser.GetOptions().GetStandardOptions();
 
+    // Call config.
+    // Put this early because it might disable the display.
+    // Also has knob setup which can affect usage message
```

Comment:
There should be a dispatch knob, since dispatch config is doing application initialization, and we really should have parsed knob values already.